Public/Src/App/Bxl/Args.cs

```diff
@@ -990,6 +990,11 @@ public bool TryParse(string[] args, PathTable pathTable, out ICommandLineConfigu
                             "unsafe_IgnoreSetFileInformationByHandle",
                             sign => sandboxConfiguration.UnsafeSandboxConfigurationMutable.IgnoreSetFileInformationByHandle = sign,
                             isUnsafe: true),
+                        OptionHandlerFactory.CreateBoolOption(
+                            "unsafe_IgnoreUndeclaredAccessesUnderSharedOpaques",

```

Comment:
Just in case this is accidental... was this change intentional? (I'm 100% fine with it, just want to double check that this change didn't sneak in by an accident).

Reply:
I reordered it alphabetically. We used to maintain that order, but the check was turned off.